src/views/ManageList.jsx

```diff
@@ -1,7 +1,71 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api';
+
+export function ManageList({ listPath }) {
+	const [item, setItem] = useState({ name: '', urgency: 'soon' });
+	const [submitted, setSubmitted] = useState();
+
+	const handleChange = (e) => {
+		setItem({ ...item, [e.target.name]: e.target.value });
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		const { name, urgency } = item;
+
+		let nextPurchasedDate;
+		switch (urgency) {
+			case 'soon':
+				nextPurchasedDate = 7;
+				break;
+			case 'kindOfSoon':
+				nextPurchasedDate = 14;
+				break;
+			case 'notSoon':
+				nextPurchasedDate = 30;
+				break;
+			default:
+				console.log('Unrecognized selection');
+				return;
+		}
+		try {
+			await addItem(listPath, {
+				itemName: name,
+				daysUntilNextPurchase: nextPurchasedDate,
+			});
+			setSubmitted('added');
+		} catch (err) {
+			console.log(err);
+			setSubmitted('failed');
+		}
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<div>
+			<p>
+				Hello from the <code>/manage-list</code> page!
+			</p>
+			{submitted === 'added' && <span>Your item was added!</span>}
+			{submitted === 'failed' && <span>Your item wasn't added!</span>}
+			<form onSubmit={handleSubmit}>
```

Comment:
I mentioned this in the other pull request, but thanks for keeping this form simple. Again, be thinking about potential styles, etc you all would want to add later.